cmd/inboxes.go

```
@@ -48,9 +53,102 @@ func handleGetInbox(r *fastglue.Request) error {
+		return
+	}
+	app.lo.Info("whatsapp webhook subscribed automatically", "inbox_id", inboxID, "callback_url", callbackURL)
+}
+
+func validateWhatsAppCredentials(r *fastglue.Request, app *App, inb imodels.Inbox) error {
+	if inb.Channel != whatsappChannel.ChannelWhatsApp || app.whatsappClient == nil {
+		return nil
+	}
+	var cfg whatsappChannel.Config
+	if err := json.Unmarshal(inb.Config, &cfg); err != nil {
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 105) but it returns nil

(nilerr)